db/migrate/20210213000000_lowercaseify_tags.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class LowercaseifyTags < ActiveRecord::Migration[5.2]
+  # We have to disable the global transaction for migrations, otherwise we
+  # can't rescue from duplicates below. Checking if there are duplicates before
+  # running the update would make this migration even more complicated, so...
+  disable_ddl_transaction!
+
+  # rubocop:disable Metrics/MethodLength
+  def up
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for up is too high. [<3, 21, 3> 21.42/20]